### PR TITLE
Adding a tv radio blog and fixing one of johns links

### PR DESCRIPTION
**Pull request by MandyMeindersma**

### CODE REVIEW

**Comment by MandyMeindersma** [comment]
I should have created this as a draft, oops. For whatever reason my local won't create the blog post, want to see if netlify will fix it for me

**Comment by MandyMeindersma** [comment]
It didn't.... I must be typing something wrong, anyone see my mistake?

**Comment by johnnewton-hub** [comment]
Maybe found your typing issue.I don't know how to mark up your code to point this out, so I'll dump it into this message-sorry. I noticed that the link to the charity shop exchange website looks off it should be (https://charityshopexchange.com) but there is some html in there.  The link is 404 

j

**Comment by MandyMeindersma** [comment]
@johnny-newton  I didn't do their website link, I was trying to link it to your Q and A blog post

**Comment by johnnewton-hub** [comment]
Yup, not a programmer, sorry it was the only thing I noticed.  I wish I could be more help here.  

j

**Comment by MandyMeindersma** [comment]
Hahaha don't be sorry! You are awesome!

**Comment by ajyong** [comment]
It's a good thing to notice and fix other bugs too :)

If no one else gets to it, I can squeeze in some lunch time to take a look
today.

**Comment by johnnewton-hub** [comment]
I'd appreciate having your eyes on it, Mandy just saved me from committing a change to one of my files on the wrong branch.

File: _posts/2020-04-07-charity-shop-exchange.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ author: John
 title: Charity Shop Exchange
 ---
 
-A large helping of thank-you to [Blake Prouty](https://twitter.com/BlakeNthaniel) for giving us insight into his team’s [Charity Shop Exchange](https://charityshopexchange.com) project. Blake is an unlikely hockey fan who hails from South Dakota, and cheers for the Calgary Flames. This is no place for hockey rivalry, so without further ado here are his thoughts on the project.
```
Comment (johnnewton-hub) [inline _posts/2020-04-07-charity-shop-exchange.md]: Hi @ajyong , I can make this change, I just don't know which branch to commit to.  Congratulations on your new gig too btw!

**Comment by johnnewton-hub** [comment]
hold off on this for a moment-I'm waiting for an address/ok from Blake-

File: _posts/2020-04-07-charity-shop-exchange.md (hunk 1 above)
Comment (johnnewton-hub) [reply _posts/2020-04-07-charity-shop-exchange.md]: here is the link that Blake would like:  https://www.linkedin.com/mwlite/in/blakenp - checked it, it works...

j-rock

**Comment by ajyong** [comment]
@MandyMeindersma it's not appearing because the date in the filename is in the future by 1 day: `2020-04-25`

When I changed it to today's date `2020-04-24` it appeared:
image

File: _posts/2020-04-25-TV+Radio.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
+author: Mandy
+title: We have been on TV and the Radio talking about how awesome our hackers are!
```
Comment (ajyong) [inline _posts/2020-04-25-TV+Radio.md]: The title is a bit long but that's okay

File: _posts/2020-04-25-TV+Radio.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+---
```
Comment (ajyong) [inline _posts/2020-04-25-TV+Radio.md]: Date in filename should be changed to `2020-04-24` for it to get picked up and rendered

**Review by ajyong — COMMENTED** [review]
LGTM after changing the filename

File: _posts/2020-04-25-TV+Radio.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
+author: Mandy
+title: We have been on TV and the Radio talking about how awesome our hackers are!
+---
+
+We are so proud of our leader Mark Bennett for talking on the Radio way back on March 16th when this thing all started! Read about the interview [here](https://www.cbc.ca/news/canada/edmonton/edmonton-programmers-hosting-virtual-covid-19-hackathon-1.5499553).
+
+We also we featured on Global Edmonton to explain a little bit about what a hackathon is! Even one of our weekly winners Blake Prouty from [Charity Shop Exchange](2020/04/07/charity-shop-exchange.html) was able to share some insights. View the story [here](https://globalnews.ca/news/6801027/codevid-19-hackathon-edmonton-coronavirus/).
```
Comment (ajyong) [inline _posts/2020-04-25-TV+Radio.md]: >We also we featured

I think this should be: We were also featured

**Review by ajyong — CHANGES_REQUESTED** [review]

**Comment by MandyMeindersma** [comment]
LOL I ACTUALLY THOUGHT IT WAS THE 25 oops. I EVEN SAID THAT IN THE MEETING LOL

**Comment by MandyMeindersma** [comment]
I have to test some of the links before it goes out, I'll do that tonight and make copy changes

**Review by ajyong — APPROVED** [review]
LGTM :shipit:

**Review by johnnewton-hub — APPROVED** [review]
If it's good enough for Aaron, it's more than good enough for me.  Squash

**Pull request merged**